mainwindow.cpp

```diff
@@ -14,12 +15,16 @@ MainWindow::MainWindow(QWidget *parent)
 //    setDarkStyle();
 
     ui->setupUi(this);
+
+    connect(&Logger::getInstance(), &Logger::logSent, this, &MainWindow::on_logSent);
+
     ui->mapWidget->setScaleFactor(0.05);
     ui->mapWidget->setSelectedObjectState(0);
     ui->mapWidget->addMapModule(ui->planUI->getRoutePlannerModule());
     ui->mapWidget->addMapModule(ui->traceUI->getTraceModule());
     ui->mapWidget->addMapModule(ui->flyUI->getGotoClickOnMapModule());
     ui->mapWidget->addMapModule(ui->cameraGimbalUI->getSetRoiByClickOnMapModule());
+    ui->logBrowser->hide();
```

Comment:
```suggestion

    Logger::initGroundStation();

    ui->setupUi(this);
    ui->logBrowser->hide();

    connect(&Logger::getInstance(), &Logger::logSent, this, &MainWindow::on_logSent);

    ui->mapWidget->setScaleFactor(0.05);
    ui->mapWidget->setSelectedObjectState(0);
    ui->mapWidget->addMapModule(ui->planUI->getRoutePlannerModule());
    ui->mapWidget->addMapModule(ui->traceUI->getTraceModule());
    ui->mapWidget->addMapModule(ui->flyUI->getGotoClickOnMapModule());
    ui->mapWidget->addMapModule(ui->cameraGimbalUI->getSetRoiByClickOnMapModule());
```

Reply:
Would suggest to execute Logger::initGroundStation(); here, move ui->logBrowser->hide(); to the setupUI call

Reply:
Suggested changes has now been implemented